src/components/PokemonDisplay.jsx

```diff
@@ -0,0 +1,128 @@
+import React, {setState, useEffect} from 'react';
+import $ from 'jquery'
+
+class Pokemon {
```

Comment:
move Pokemon to its own Pokemon.jsx file.  this will make it so you can include it anywhere separately